### PR TITLE
Add live latency test to the index page

### PR DESCRIPTION
**Pull request by evgenykuzyakov**
Add live latency test. Refactor index.html out
Screen Shot 2024-10-11 at 11 34 35 AM

### CODE REVIEW

File: static/index.html [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        }
+      }
+    };
+    fetcher().then(() => {
+    });
+
+  </script>
+</div>
+
+<h2>API</h2>
```
Comment (mikedotexe) [inline static/index.html:106]: Solid info here 💪🏻

File: static/index.html [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+  </style>
+</head>
+<body>
+<h1>NEAR Data Server by FASTNEAR</h1>
+<p>For more information, visit <a href='https://github.com/fastnear/neardata-server/'>GitHub</a></p>
+
+<h2>LIVE LATENCY TEST</h2>
+<div id="latency">
+
+  <script src="https://cdn.jsdelivr.net/npm/chart.js@3.7.1/dist/chart.min.js"></script>
```
Comment (mikedotexe) [inline static/index.html:30]: Haven't seen this in a while, and forgot how simple and clean it is. If it's able to pull in any ol' static HTML file, I'm already thinking of how these might be generated/updated on the server. /hack-it-friday-thought

**Review by mikedotexe — APPROVED** [review]
looks great, and good to see how easy it is to add new visualization pages. ones that aren't tied to a framework but can be anything we want

File: static/index.html (hunk 2 above)
Comment (evgenykuzyakov) [reply static/index.html:30]: This basic chart example was actually generated by demo.fastnear.ai 
Then just modified it to feed real data

**Pull request merged**